platform/src/EducationPlatformApp.js

```diff
@@ -152,7 +155,7 @@ class EducationPlatformApp {
 
         if (errors.length==0){
             // An activity configuration has been provided
-            this.toolsManager = new ToolsManager(this.errorNotification);
+            this.toolsManager = new ToolsManager(this.errorHandler.notify.bind(this.errorHandler));
```

Comment:
This is quite complicated "magic". Is there a way of simplifying this?

Reply:
@szschaler the `errorNotification()` etc could be moved to `PlaygroundUtility.js` and imported as required from there and means that it doesn't have to be passed into `ErrorHandler`

Reply:
Simplified by moving notification functions to `PlaygroundUtility` in bbabad1ec6f1ea7a340b766d1c5ce449b0c08ed7. 

Rename `PlaygroundUtility` as a separate task